.github/workflows/configlet-sync.yml

```
@@ -0,0 +1,93 @@
+name: Configlet Auto Sync
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 15 * *'
+
+jobs:
+  sync-docs-metadata:
+    timeout-minutes: 10
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fetch configlet
+        run: ./bin/fetch-configlet
+
+      - name: Run configlet sync for files
+        run: ./bin/configlet sync --docs --metadata --filepaths -u -y
+
+      - name: Create pull request if changes
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "🤖 Auto-sync docs, metadata, and filepaths"
+          title: "🤖 Configlet sync: docs, metadata, and filepaths"
+          body: |
+            This PR was generated automatically by a scheduled workflow.
+
+            It includes updates from `configlet sync` for:
+            - 📄 Documentation
+            - 🧭 Metadata
+            - 🗂️ Filepaths
+
+            Please review and merge if everything looks good!
+          branch: configlet-auto-sync
+          delete-branch: true
+
+  check-test-sync:
+    timeout-minutes: 10
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fetch configlet
+        run: ./bin/fetch-configlet
+
+      - name: Run configlet sync for test and capture output
+        id: sync_test
+        run: |
+          configlet_raw_output="$(./bin/configlet sync --tests | tee .github/sync-test-output.txt)"
+
+          echo "configlet output:"
+          echo "$configlet_raw_output"
+
+          echo '```' > .github/sync-test-output.txt
+          echo "$configlet_raw_output" >> .github/sync-test-output.txt
+          echo '```' >> .github/sync-test-output.txt
+
+          echo "output<<EOF" >> "$GITHUB_OUTPUT"
+          echo "$configlet_raw_output" >> "$GITHUB_OUTPUT"
+          echo "EOF" >> "$GITHUB_OUTPUT"
```

Is there a reason the .github directory would be used as a scratch location? That path should be used for github related configurations. Usually /tmp or a tempfile is used for this sort of thing.

Additionally, you write the contents to this file (L55) then immediately overwrite the contents a few lines later (L60).

```suggestion
          configlet_raw_output="$(./bin/configlet sync --tests)"

          printf "configlet output:\n%s\n" "$configlet_raw_output"
          printf '```\n%s\n```\n' "$configlet_raw_output" > .github/sync-test-output.txt
          printf 'output<<EOF\n%s\nEOF\n' "$configlet_raw_output" > "$GITHUB_OUTPUT"
```

Is the output needed in two files?

---

> Additionally, you write the contents to this file (L55) then immediately overwrite the contents a few lines later (L60).

You're absolutely right, I had meant to update line 55 so it only saved the variable, but I missed it. Thanks for catching that!

Also, really appreciate the other suggestions. They’ve definitely helped clean up and shorten the code. I agree that using a `temp` folder makes more sense here, so I’ll go ahead and make that change as well.